.lando.yml

```diff
@@ -0,0 +1,38 @@
+name: drupal-scaffold
+recipe: drupal8
+
+config:
+  database: mariadb:10.1
+  php: "7.1"
+  webroot: web
+  xdebug: true
+
+events:
+  # Clear cache after a database import
+  post-db-import:
+    - appserver: cd $LANDO_WEBROOT && drush cache-rebuild --yes
+
+services:
+  appserver:
+    run:
+      - cd $LANDO_MOUNT && composer install
+
+  nodejs:
+    type: node:8.9
+    globals:
+      gulp-cli: latest
+    run:
+      - cd $LANDO_MOUNT && npm install
+
+  phpmyadmin:
+    type: phpmyadmin
```

Comment:
Don't you need the environment variables here? or does it work properly now

Reply:
Works finally! 🙌

Reply:
Noice